code/modules/antagonists/blob/blob_abilities.dm

```diff
@@ -474,7 +476,7 @@
 		if (!tutorial_check("attack", T))
 			return
 
-
+		owner.playsound_local(owner.loc, "sound/voice/blob/blob[pick("deploy", "attack")].ogg", 50, 1)
```

Comment:
why is attack playing deploy sounds

Reply:
Because deploy was originally made to be attack but I decided to make it deploy. Since you want me to be as specific as possible stuff like this happens.

Reply:
you can make more overarching categories that can describe both deploying and attacking for example.... blobaction or something for example